elastic/integrations
# [elastic_agent] Add back apm-server metrics dropped due to TSDB

## Proposed commit message

[elastic_agent] Add back apm-server monitoring metrics dropped due to TSDB

- apm-server metrics were dropped due to TSDB change in https://github.com/elastic/integrations/pull/7214
- `output` field depends on https://github.com/elastic/beats/pull/40127



## Checklist


## Author's Checklist



## How to test this PR locally


- Spin up local elastic-package stack with changes
- Start EA with agent monitoring enabled
- Add APM to EA
- Confirm that apm-server monitoring metric fields are stored

## Related issues

Part of https://github.com/elastic/apm-server/issues/13475


- 

## Screenshots
